tests/e2e/vLLM/run_tests.sh

```diff
@@ -5,26 +5,30 @@ SUCCESS=0
 # Parse list of configs.
 MODEL_CONFIGS="$PWD/tests/e2e/vLLM/configs"
 
-for MODEL_CONFIG in "$MODEL_CONFIGS"/*
-do
+# Function to process a file
+process_model_config() {
+    local MODEL_CONFIG=$1
     LOCAL_SUCCESS=0
-
+    
     echo "=== RUNNING MODEL: $MODEL_CONFIG ==="
-
-    export TEST_DATA_FILE="$MODEL_CONFIG"
-    pytest \
-        --capture=tee-sys \
-        --junitxml="test-results/e2e-$(date +%s).xml" \
-        "$PWD/tests/e2e/vLLM/test_vllm.py" || LOCAL_SUCCESS=$?
-
+    export TEST_DATA_FILE=${MODEL_CONFIG}
+    pytest -s $PWD/tests/e2e/vLLM/test_vllm.py || LOCAL_SUCCESS=$?
```

Comment:
This change to the `pytest` command is unnecessary and removes important, required flags that were recently added.

Reply:
Ok